tests/bearlib/aspects/ModuleTest.py

```diff
@@ -19,10 +19,12 @@ def test_module(self):
 
     def test__getitem__(self):
         # check a leaf aspect
-        for aspectname in ['aspectsYEAH', 'spelling.aspectsYEAH',
-                           'root.SPELLING.aspectsYEAH']:
-            assert (coalib.bearlib.aspects[aspectname] is
-                    coalib.bearlib.aspects.Root.Spelling.aspectsYEAH)
+        for aspectname in ['GeneralWordSpelling',
+                           'spelling.GeneralWordSpelling',
+                           'root.SPELLING.GeneralWordSpelling']:
+            assert (
+                coalib.bearlib.aspects[aspectname] is
+                coalib.bearlib.aspects.Root.Spelling.GeneralWordSpelling)
```

Comment:
put `coalib.bearlib.aspects.Root.Spelling.GeneralWordSpelling` in a variable so you can remove the round brackets.

And if possible, replace `assert` with `self.assertIs`